components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -62,6 +62,7 @@
 import org.wso2.carbon.apimgt.api.APIManagementException;
```

Comment:
This host object class is reformatted.

Reply:
It's fine :+1:

Reply:
But normally when you do a feature please format your code firstt. In a separate PR you can fix formatting issues of other code which will be easy for reviewers to go through your changes.

Reply:
Noted.